.travis.yml

```diff
@@ -14,7 +14,7 @@ script:
 
   # * E501 - line length limit
   # * E123 - indentation on data structures
-  - flake8 . --ignore=F403,E501,E123,E128 --exclude=docs/
+  - flake8 . --ignore=F403,E501,E123,E128,F401 --exclude=docs/
```

Comment:
This is problem. This is because I am importing into the __init__.py files in order to have them covered by tests. However, then it returns errors on flake8. I am not sure how resolve. For now, I put this in. Suggestions? If you run locally, you can see what it returns: `flake8 . --ignore=F403,E501,E123,E128 --exclude=docs/`